### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by Harrison-Salzverg**
This PR provides several updates to the documentation, including download instructions, client requirements, additional contribution credits, and a CONTRIBUTING.md file which provides contribution guidelines.  Also added a period ;)

### CODE REVIEW

**Comment by jackconryan** [comment]
Out of office until 9th October

**Review by LukeBogdanovic — COMMENTED** [review]
This PR doesn't follow the needs set out by the published issue. It may also be no harm to link the issue and the PR to each other as well as adding label/s showing what this PR updates at a glance, in this case documentation. It would also be good to assign this PR to someone and request review of the PR.

The corresponding issue is talking about **updating the documentation related to the code within the Python file/s**. Please read what the issue states carefully before contributing.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Contributing Guidelines
+1. Create a new branch for your contribution.
+2. Make sure your branch is up-to-date: `git pull upstream main`
+3. Make your changes.
+5. Push your changes to your fork.
+6. Squash commits before making a PR.
+7. Submit a pull request through GitHub.
```
Comment (jackconryan) [inline CONTRIBUTING.md:7]: The issue: https://github.com/jackconryan/DartsElo/issues/5#issue-1915982105 is not referencing CONTRIBUTION.md or README.md documentation. it is referencing out of date documentation in the code. 

I'm closing this PR as it seems there is a lack of understanding in the code (see 'npm install' instruction in python project) and lack of understanding of the issue

File: README.md [hunk 2]
```diff
@@ -4,11 +4,29 @@ Creating an Elo system for our local Darts matches/tournaments.
 The Aim with this project is to creating a functional Elo system similar to one which can be found in games such as chess and Go for Darts.
 The product must be easy to use and accessible for non technical users for use in our local darts matches which take place during our tournaments. 
 
-Special thanks to Daniel Ryan for providing the R code and mathematical expertisehis original product can be seen in the ExcelSheet.R file and Coolmoyne Leaderboards.xlsx excel file. 
+## Download instructions
+Run `git clone https://github.com/jackconryan/DartsElo`
 
+Run `npm install` to download dependencies
```
Comment (jackconryan) [inline README.md:10]: npm is not relevant to this project it is used for Javascript and node.js which is not present in the code in its current state.

**Pull request closed without merging**